.changeset/cyan-bugs-reflect.md

```diff
@@ -0,0 +1,10 @@
+---
+'@shopify/hydrogen-react': patch
+'customer-api': patch
+'@shopify/mini-oxygen': patch
+'multi': patch
```

Comment:
@juanpprieto changed the name of `multi` to `multipass` in https://github.com/Shopify/hydrogen/pull/1540/files
but I'm not sure if that also changes its name for changesets?

Reply:
possible! I will rebase and run changeset again to see